README.md

```diff
@@ -1,14 +1,23 @@
-Backdrop Contributed Project Group Application
+What is the difference between Backdrop Contrib Project Group and backdrop-ops/contrib?
```

Comment:
I don't like questions in headers unless they are rhetorical. I like "An Introduction To The Backdrop Contrib Project Group and backdrop-ops/contrib Repository".